docs/fig.md

```diff
@@ -0,0 +1,44 @@
+# Fig Auto-complete
```

Comment:
Maybe the filename is better to use "fig-autocomplete.md", for many users know the "autocomplete", but don't know the fig. (maybe, just my personal guess)

Reply:
I agree. Maybe simply "autocomplete" without even mentioning "fig" in the doc's filename/title is also fine. Then in the first paragraph, we can mention that it's implemented by using xxx tool.

Reply:
I will rename it to autocomplete and also mention other complete tools. WDYT?

Reply:
> I agree. Maybe simply "autocomplete" without even mentioning "fig" in the doc's filename/title is also fine. Then in the first paragraph, we can mention that it's implemented by using xxx tool.

autocomplete is better. because we could add "linux support with other tool" in the same doc.